changes/ce/fix-14001.en.md

```diff
@@ -0,0 +1 @@
+Fixed a race condition where a resource (connector/action/source/authentication/authorization) could report having a connected, healthy channel where in actuality it was broken, after a moment of disconnection.
```

Comment:
maybe describe what errors will be observed due to this error.
and also add it to known issues: https://docs.emqx.com/en/emqx/latest/changes/known-issues-5.8.html#known-issues-in-emqx-5-8

```suggestion
Fixed a race condition where a resource (connector/action/source/authentication/authorization) could report having a connected, healthy channel where actuality it was broken, after a moment of disconnection.
```

Reply:
I think the expression is "where in actuality", not "where actuality".  :thinking: